circle.yml

```diff
@@ -23,3 +23,4 @@ test:
     - npm run citest-jasmine
     - npm run test-bundle
     - npm run test-syntax
+    - eslint .
```

Comment:
Note that `npm run lint` wouldn't work because of the [`|| true`](https://github.com/plotly/plotly.js/blob/fbc7597407c472afa0abe1993c4d3fadaa384dfd/package.json#L30) fall back in the npm script. 

This fall back removes the ugly npm errors when eslint detects errors but also make Circle think that linting errors aren't failed tests.